src/containers/PermissionsApplication.jsx

```diff
@@ -1,25 +1,63 @@
 import React from 'react'
 
 import { APPS_DOCTYPE } from 'doctypes'
-import { Q, useQuery, isQueryLoading, hasQueryBeenLoaded } from 'cozy-client'
 import Spinner from 'cozy-ui/transpiled/react/Spinner'
 import Typography from 'cozy-ui/transpiled/react/Typography'
 import { useI18n } from 'cozy-ui/transpiled/react'
 import Page from 'components/Page'
 import PageTitle from 'components/PageTitle'
 import { withRouter, Link } from 'react-router-dom'
+import {
+  Q,
+  useQuery,
+  isQueryLoading,
+  hasQueryBeenLoaded,
+  models
+} from 'cozy-client'
+
+const {
+  locales: { getBoundT }
+} = models.document
+
+export const completePermission = (key, permission, value) => {
```

Comment:
```suggestion
export const completePermission = (name, permission, { description, verbs }) => {
```

Reply:
and replace `value.description` by `description` + replace `value.verbs` by `verbs`

Reply:
and replace `key`by `name` below